### PR TITLE
US13319: Music Landing Page

### PR DESCRIPTION
**Pull request by lukemoderwell**
- Music landing page markup/styles
- migration scripts for albums/songs
- config changes
- add crds-music logo and new instagram logo to icons.svg

### CODE REVIEW

File: music.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+worship_training_cta: Learn More
+---
+
+{% assign sorted_songs = site.songs | reverse %}
+
+<div class="container">
+  <p class="breadcrumb breadcrumb-media flush-bottom hard"><a href="{{ site.url }}">Media</a> /</p>
+  <h2 class="section-header landing-header flush-top">Music</h2>
+  {% assign hero = sorted_songs | first %}
+  <div class="jumbotron jumbotron-xl" style="background-image: url('{{ hero.image }}">
```
Comment (tcmacdonald) [inline music.html]: Should `{{ hero.image }}` be piped to the imgix filter?

File: music.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  </div>
+
+  <div class="card-deck">
+    <div class="cards-2x">
+      <div class="row">
+        {% assign featured = sorted_songs | slice: 1, 2 %}
+        {% for song in featured %}
+          <div class="card">
+            <a href="{{ song.url }}">
+              <img class="card-img-unrestrained" src="{{ song.image }}">
```
Comment (tcmacdonald) [inline music.html]: Same comment here... `{{ song.image }}` should implement the imgix filter.

File: _layouts/song.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+layout: default
+---
+
+Songs!
```
Comment (tcmacdonald) [inline _layouts/song.html]: Is this file going to be populated in a future story?

File: _migrations/06-create-albums.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+module.exports = function (migration) {
```
Comment (seancdavis) [inline _migrations/06-create-albums.js]: There is already a 06 migration.

File: music.html (hunk 1 above)
Comment (seancdavis) [reply music.html]: Also looks like there's a syntax error here. Missing a `'` and `)` (and optionally, a `;`).

File: _migrations/06-create-albums.js (hunk 4 above)
Comment (lukemoderwell) [reply _migrations/06-create-albums.js]: must have been added via a development branch merge. Will change.

**Pull request merged**